wowup-electron/src/app/pages/my-addons/my-addons.component.html

```diff
@@ -227,6 +227,9 @@ <h1>{{ "COMMON.SEARCH.NO_ADDONS" | translate }}</h1>
     <button mat-menu-item (click)="onShowFolder(listItem.addon)">
       {{ "PAGES.MY_ADDONS.ADDON_CONTEXT_MENU.SHOW_FOLDER" | translate }}
     </button>
+    <button mat-menu-item (click)="onVisitProvider(listItem.addon)">
+      {{ "DIALOGS.ADDON_DETAILS.VIEW_ON_PROVIDER_PREFIX" | translate }} {{listItem.addon.providerName}}
```

Comment:
Can we add the same little icon from details for consistency? 

https://github.com/WowUp/WowUp/blob/develop/wowup-electron/src/app/components/addon-detail/addon-detail.component.html#L36

Reply:
icon is added, hope that I resized it appropriately in the css